packages/squiggle-lang/src/rescript/Utility/E/E_A.res

```diff
@@ -272,7 +270,7 @@ module Floats = {
       ~minDiscreteWeight: int,
     ) => {
       let continuous: array<float> = []
-      let discrete = FloatFloatMap.empty()
+      let discrete: array<(float, int)> = []
```

Comment:
Just curious; how much do you think this speeds it up?

Reply:
Don't care. If you're never going to look up a value, storing things in a map doesn't convey the purpose of the code well. The actual answer could easily vary by a factor of 10 depending on how the map is implemented and I haven't benchmarked it.